FurEver/wwwroot/css/quiz.css

```diff
@@ -0,0 +1,82 @@
+﻿﻿ /* quiz.css */
+.container {
+    max-width: 600px;
+    margin: auto;
+    text-align: center;
+    font-family: Arial, sans-serif;
+}
+
+h1 {
+    font-size: 2em;
+    margin-bottom: 20px;
+    color: #333;
+}
+
+.dog-image {
+    margin-bottom: 20px;
+}
+
+    .dog-image img {
+        width: 50%;
+        height: 50%;
+        border-radius: 8px;
+        box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    }
+
+.quiz-form {
+    background-color: #f8f9fa;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+    .quiz-form p {
+        font-size: 1.2em;
+        color: #555;
+        margin-bottom: 15px;
+    }
+    
+.options-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: 15px;
+}
+
+.option-label {
+    display: flex;
+    align-items: center;
+    font-size: 1em;
+    color: #333;
+    margin: 5px 0;
+}
+
+option-label input[type="radio"] {
+    margin-right: 10px;
+}
+
+.quiz-result {
+    margin-top: 20px;
+    font-weight: bold;
+    color: #28a745;
+}
+
+.quiz-result-wrong {
```

Comment:
# Technical Change #2
## Allowing for an incorrect quiz answer to be shown in red instead of green
This change reduces technical debt by improving the accessibility and clarity of the user-interface, aligning it with WCAG 2.1 guideline 1.4.1 Use of Color. By allowing incorrect quiz answers to be displayed in red instead of green, users can easily distinguish between correct and incorrect responses without relying solely on color differentiation, which can be problematic for individuals with color vision deficiencies. This modification ensures that feedback is more universally understood and accessible to all users, enhancing the overall user experience. Proactively addressing accessibility needs minimizes the risk of future rework to make the application compliant with accessibility standards. Moreover, this change reduces user confusion and potential errors, making the quiz interaction more intuitive and robust, ultimately leading to a more maintainable and user-friendly codebase. By implementing this enhancement, you also ensure better compliance with accessibility requirements, decreasing the likelihood of needing extensive accessibility audits and fixes in the future.
Link to Reference: https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html